rest/route/service_test.go

```diff
@@ -1,1257 +1,1327 @@
-		DisplayOnly:    true,
-		ExecutionTasks: []string{"execTask0", "execTask1"},
-	}
-	execTask0 := task.Task{
-		Id:      "execTask0",
-		BuildId: buildID,
-	}
-	execTask1 := task.Task{
-		Id:      "execTask1",
-		BuildId: buildID,
-	}
-	randomTask := task.Task{
-		Id:      "randomTask",
-		BuildId: buildID,
-	}
-
-	assert.NoError(t, displayTask.Insert())
-	assert.NoError(t, execTask0.Insert())
-	assert.NoError(t, execTask1.Insert())
-	assert.NoError(t, randomTask.Insert())
-	ctx := context.Background()
-	tbh := &tasksByBuildHandler{
-		limit: 100,
-		sc:    &sc,
-	}
-	route := "/rest/v2/builds/test/tasks?fetch_all_executions=false&fetch_parent_ids=true&start_at=execTask0"
-	r, err := http.NewRequest("GET", route, nil)
-	assert.NoError(t, err)
-	r = gimlet.SetURLVars(r, map[string]string{"build_id": "test"})
-
-	assert.NoError(t, tbh.Parse(ctx, r))
-	assert.False(t, tbh.fetchAllExecutions)
-	assert.True(t, tbh.fetchParentIds)
-
-	resp := tbh.Run(ctx)
-	data, ok := resp.Data().([]interface{})
-	assert.True(t, ok)
-	assert.Equal(t, 3, len(data))
-
-	expectedParentIDs := []string{dtID, dtID, ""}
-	for i := range data {
-		task, ok := data[i].(*model.APITask)
-		require.True(t, ok)
-		assert.Equal(t, expectedParentIDs[i], task.ParentTaskId)
-	}
-}
-
-func TestOptionsRequest(t *testing.T) {
-	assert.NoError(t, db.ClearCollections(task.Collection))
-
-	route := "/rest/v2/tasks/test/restart"
-	_, err := http.NewRequest("OPTIONS", route, nil)
-	assert.NoError(t, err)
-	ctx := context.Background()
-
-	tbh := &optionsHandler{}
-	resp := tbh.Run(ctx)
-	data, ok := resp.Data().(interface{})
-	assert.True(t, ok)
-	assert.Equal(t, data, struct{}{})
-	assert.Equal(t, resp.Status(), http.StatusOK)
-
-}
-
-func validatePaginatedResponse(t *testing.T, h gimlet.RouteHandler, expected []model.Model, pages *gimlet.ResponsePages) {
-	if !assert.NotNil(t, h) {
-		return
-	}
-	if !assert.NotNil(t, pages) {
-		return
-	}
-
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
-	resp := h.Run(ctx)
-	assert.NotNil(t, resp)
-	assert.Equal(t, http.StatusOK, resp.Status())
-
-	rpg := resp.Pages()
-	if !assert.NotNil(t, rpg) {
-		return
-	}
-
-	assert.True(t, pages.Next != nil || pages.Prev != nil)
-	assert.True(t, rpg.Next != nil || rpg.Prev != nil)
-	if pages.Next != nil {
-		assert.Equal(t, pages.Next.Key, rpg.Next.Key)
-		assert.Equal(t, pages.Next.Limit, rpg.Next.Limit)
-		assert.Equal(t, pages.Next.Relation, rpg.Next.Relation)
-	} else if pages.Prev != nil {
-		assert.Equal(t, pages.Prev.Key, rpg.Prev.Key)
-		assert.Equal(t, pages.Prev.Limit, rpg.Prev.Limit)
-		assert.Equal(t, pages.Prev.Relation, rpg.Prev.Relation)
-	}
-
-	assert.EqualValues(t, pages, rpg)
-
-	data, ok := resp.Data().([]interface{})
-	assert.True(t, ok)
-
-	if !assert.Equal(t, len(expected), len(data)) {
-		return
-	}
-
-	for idx := range expected {
-		m, ok := data[idx].(model.Model)
-
-		if assert.True(t, ok) {
-			assert.Equal(t, expected[idx], m)
-		}
-	}
-}
+//
+//import (
```

Comment:
Leftover testing? :)

Reply:
Yes this file in particular had been difficult to get to pass so I had it commented out while I work on them as the last failing tests. Uncommented now